src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -9,7 +9,18 @@
       <v-card flat class="mt-5">
         <NameRequestInfo />
 
-        <!-- FUTURE: Business Type goes here -->
+        <v-divider class="mx-6" />
+
+        <!-- Business Type -->
+        <header id="business-type-confirm-header" />
+        <BusinessTypeConfirm
+          class="py-8 px-6"
+          :class="{ 'invalid-section': getShowErrors && !businessTypeConfirmValid }"
+          :businessTypeConfirm="getRegistration.businessTypeConfirm"
+          :isTypePartnership="isTypePartnership"
+          @update:businessTypeConfirm="setRegistrationBusinessTypeConfirm($event)"
+          @valid="onBusinessTypeConfirmValidEvent($event)"
+        />
```

Comment:
In the latest commit 17a9bb1, I removed the template tag and v-if condition, because we will the checkbox for SP too. This is the [design link](https://projects.invisionapp.com/share/3511UPXPMT2P#/screens/460287155_Step_1-2_SP_-_DBA_Errors).